qmri_vfa/sub-01/anat/sub-01_flip-1_VFA.json

```diff
@@ -1,6 +1,7 @@
 {
   "IntendedFor": [
-    "anat/sub-01_T1map.nii.gz"
+    "anat/sub-01_T1map.nii.gz",
+    "anat/sub-01_M0map.nii.gz"
   ],
```

Comment:
This is not defined for anatomicals, and pointing from a raw dataset to a derivative dataset is not supported. (IntendedFor is always within a subject directory as currently defined.)

Reply:
related https://github.com/bids-standard/bids-specification/issues/718

Reply:
I was wondering if this was possible, thanks for the clarification @effigies!